Otherwise, the tests don't build with -buildmode=pie.

See: #15809

Alternative solution: Build tests with -buildmode=default.

diff -upr doctl-1.104.0.orig/integration/init_test.go doctl-1.104.0/integration/init_test.go
--- doctl-1.104.0.orig/integration/init_test.go	2024-03-11 11:51:13.713693546 +0100
+++ doctl-1.104.0/integration/init_test.go	2024-03-11 11:52:19.110550884 +0100
@@ -42,7 +42,6 @@ func TestMain(m *testing.M) {
 
 	// tried to use -mod=vendor but it blew up
 	cmd := exec.Command("go", "build", "-o", builtBinaryPath, packagePath)
-	cmd.Env = append(os.Environ(), "CGO_ENABLED=0")
 	output, err := cmd.CombinedOutput()
 	if err != nil {
 		panic(fmt.Sprintf("failed to build doctl: %s", output))
